app/controllers/user_controller.rb

```diff
@@ -712,6 +713,7 @@ def update_user(user, params)
     end
 
     if user.save
+      @preferred_languages = current_user.preferred_languages
```

Comment:
What is this doing? Does anything actually use that member of the controller?

Was it an early attempt to reset the cache on the model that has since been replaced with the change in the model?

Reply:
It was a hack, that in combination with the removed memoization in `user.rb` resulted in the preferred_languages being recomputed